crates/ev-precompiles/README.md

```
@@ -190,3 +190,63 @@ cast send --rpc-url $RPC_URL --private-key $OPERATOR_KEY \
   "mint(address,uint256)" 0xRECIPIENT 1000000000000000000
 ```
+
+## Proposer Control Precompile
+
+The proposer control precompile stores the ev-node proposer that should sign the next block. It is
+used by the ev-node EVM execution adapter to populate ADR-023's `NextProposerAddress` from execution
+state.
+
+### Address
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)